config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
   get '/restaurant', controller: :restaurant, action: :index
+
+  resources :dishes, only: [:new, :show, :create, :edit, :destroy]
```

Comment:
Please just create the things needed atm. Really, if you create other routes you deprive whoever will work on the next feature the joy of creating them. 😉😀